src/veh_interact.cpp

```diff
@@ -847,15 +847,34 @@ bool veh_interact::update_part_requirements()
         ok = false;
     }
 
+    std::string str_suffix;
+    int adjusted_strength = player_character.get_str();
+    if( player_character.has_trait( trait_id( "STRONGBACK" ) ) ) {
+        adjusted_strength *= 1.35;
+    } else if( player_character.has_trait( trait_id( "BADBACK" ) ) ) {
+        adjusted_strength /= 1.35;
+    }
```

Comment:
I recommend extract the calculation of `STRONGBACK` or `BADBACK` adjusted strength to a helper member function of `player` class so there won't be duplicate code, and avoids inconsistency if someone changes this adjustment factor in one place in the future but forgets other two places.

Reply:
good idea, should be updated now